README.md

```diff
@@ -1,27 +1,50 @@
 # Memory Inspector
 
-This extension contributes a set of widgets for viewing memory in different ways.
+A Visual Studio Code extension that provides a powerful and configurable memory viewer that works with debug adapters.
 
-## Requirements
+![Screenshot of the Memory Inspector](https://raw.githubusercontent.com/eclipse-cdt-cloud/vscode-memory-inspector/master/media/memory-inspector-screenshot.png)
 
-This extension must be used in conjunction with a Debug Adapter that implements a ReadMemoryRequest handler or alternative custom request that returns
-memory data. It has been tested against the [CDT-GDB Adapter](https://github.com/eclipse-cdt/cdt-gdb-adapter) used as the backend for the 
-[CDT-GDB VSCode](https://github.com/eclipse-cdt/cdt-gdb-vscode) plugin.
+## Features
 
-## The Memory Provider
+- **Multiple Format Display**: Shows memory data in different formats, provided your debug adapter supports the `ReadMemoryRequest`.
```

Comment:
`Multiple Format`: I don't think we have that yet in the data contents area. Maybe name it `Configurable Memory Display` and mention the `Multiple Format Display` for hover-over?